documentation/nn/nn.md

```diff
@@ -7,36 +7,87 @@ weight: 0
+<a href="fig/doi_nickname_window.png" data-toggle="lightbox"><img src="fig/doi_nickname_window.png" class="img-responsive" alt="How to load a Bioimage Model Zoo model in ilastik by copying either nickname or doi."/></a>
+
+Then in ilastik, paste the doi, or nickname to the model text field (1) and press the arrow-button (2).
+This will initiate the download and initialization of the network.
+
+<a href="fig/load_model_01.png" data-toggle="lightbox"><img src="fig/load_model_01.png" class="img-responsive" alt="Screenshot that illustrates loading a pre-trained neural network from the Bioimage Model Zoo by specifying the doi or the nickname of the model."/></a>
 
-3. Only one thing left to do: press the "Live Predict" button to make the network predict:
+Alternatively, you can download a zip archive of the model by clicking on "Download (format)".
+The server will then package the model with the correct weight format for ilastik consumption.  
+Once the file is downloaded you can either drag and drop the model zip file from your file explorer window into the text field, or click the arrow button while the text field is empty to bring up a load file dialog.
+
+### 3. Run the Network
+
+Only one thing left to do: press the "Live Predict" button to make the network predict:
 <a href="fig/predicted.png" data-toggle="lightbox"><img src="fig/predicted.png" class="img-responsive" /></a>
 
-4. If you like the results, proceed to export them in the [Data Export applet]({{site.baseurl}}/documentation/basics/export). If you don't, unload this model by pressing the red cross and try out another one. 
+If you prefer a different color for the display of the predictions you can right click on the corresponding layer [in the layerstack]({{site.baseurl}}/documentation/basics/layers) and select _Change color_.
+
+
+### 4. Export the Result
+
+If you like the results, proceed to export them in the [Data Export applet]({{site.baseurl}}/documentation/basics/export). If you don't, unload this model by pressing the red cross and try out another one. 
+
+## _Neural Network Classification (remote)_
+
+When running remotely, the heavy lifting of neural network computations will happen on a remote machine, while you are loading the data and selecting the network on your local machine.
+
+### On the remote machine
+
+1. Make sure the tiktorch server and neural network framework you want to run are installed via mamba.
+   Installation instructions can be found [in the tiktorch github repository](https://github.com/ilastik/tiktorch#installation).
+2. Activate the tiktorch environment and start the server giving the `--addr 0.0.0.0` flag to make it accessible.
+3. Take note of the ip/hostname of the remote machine, and the port (what out for the `Starting server on 0.0.0.0:<port>` output).
+
+### On your local machine/laptop
+
```

Comment:
Why would you want to run the remote-workflow with the tiktorch server on your own machine (as a use case rather than for dev purposes)?

Reply:
Okay, the docs are supposed to describe what you have to do to use the remote workflow. For this you have to a) setup the server on the remote machine, and b) run the remote workflow in ilastik on your local machine. Does this make sense?